resources/resolver.go

```diff
@@ -801,11 +801,32 @@ func ResolveConfig(vocabId string, token string) (config *HFConfig,
 // Given a set of resources, resolve the HuggingFace configuration.
 // Used to be able to resolve both embedded and local resources.
 func ResolveHFFromResources(resources *Resources, hfConfig *HFConfig) (*HFConfig, error) {
-	// Use interfaces to unmarsal the config file and tokenizer config file
+	// Resolve config and tokenizer config from resources
+	// config.json and tokenizer_config.json
+	hfConfig, err := ResolveConfigAndTokenizerConfig(resources, hfConfig)
+	if err != nil {
+		return nil, err
+	}
+
+	// Resolve special tokens and special tokens config from resources
+	// special_tokens_map.json and specials.txt
+	hfConfig, err = ResolveSpecialsAndSpecialTokens(resources, hfConfig)
```

Comment:
If these functions are solely helper functions for `ResolveHFFromResources()` and are not exported, they should be `camelCase`.

Reply:
ok makes sense, let me change that!